src/controllers/lift.ts

```diff
@@ -0,0 +1,98 @@
+import express, { Request, Response, NextFunction } from "express";
+import lift, { ILift } from "../models/lift";
+import { EStatus } from "../constants/enums";
+import LiftModel from "../models/lift";
+import mongoose from "mongoose";
+import Joi from "joi";
+
+const LiftValidation = Joi.object({
+  name: Joi.string().required(),
+  elevation_gain: Joi.number().min(1).max(100).required(),
+  status: Joi.string().valid("HOLD", "OPEN", "CLOSED").required(),
+});
+
+export const CreateLift = async (
+  req: Request,
+  res: Response,
+  next: NextFunction
+) => {
+  try {
+    const { error, value } = LiftValidation.validate(req.body);
+    if (error) {
```

Comment:
Remove branching.... errrrrrr... 
create validation folder and throw error from that file